sites/geohub/src/routes/(app)/data/upload/+page.svelte

```diff
@@ -1,29 +1,55 @@
 <script lang="ts">
 	import { enhance } from '$app/forms';
 	import { goto } from '$app/navigation';
-	import FieldControl from '$components/util/FieldControl.svelte';
 	import Notification from '$components/util/Notification.svelte';
 	import { AccepedExtensions } from '$lib/config/AppConfig';
 	import { BlockBlobClient } from '@azure/storage-blob';
+	import { TextInput, Checkbox } from '@undp-data/svelte-undp-design';
+	import JSZip from 'jszip';
 	import { toast } from '@zerodevx/svelte-toast';
 	import { filesize } from 'filesize';
 	import Dropzone from 'svelte-file-dropzone/Dropzone.svelte';
 	import isValidFilename from 'valid-filename';
 	import type { PageData } from './$types';
+	import Time from 'svelte-time';
+	import FieldControl from '$components/util/FieldControl.svelte';
+	import Help from '$components/util/Help.svelte';
 
-	const REDIRECRT_TIME = 2000; // two second
+	const REDIRECT_TIME = 2000; // two second
 	const FILE_SIZE_THRESHOLD = 104857600; // 100MB
-
+	let no_show_extensions = [
```

Comment:
Use this from `sites/geohub/src/lib/config/AppConfig/AcceptedExtensions.ts`
It’s duplicated